roles/kubernetes-apps/meta/main.yml

```diff
@@ -17,3 +17,5 @@ dependencies:
   - role: kubernetes-apps/efk
     when: efk_enabled
     tags: [ apps, efk ]
+  - role: kubernetes-apps/heapster
```

Comment:
this should not be automatically installed. not everyone needs it. we need a condition somewhere for heapster_enabled